--- setup.py.bak	2012-01-19 16:30:32.432816914 -0500
+++ setup.py	2012-01-19 16:37:26.448942795 -0500
@@ -376,7 +376,7 @@
                        ],
             include_dirs = [os.path.join(package_prefix,
                                          'rpy', 'rinterface'),] + include_dirs,
-            libraries = ['R', ],
+            libraries = ['R', 'readline', ],
             library_dirs = r_libs,
             define_macros = define_macros,
             runtime_library_dirs = r_libs,
